measurement/signals/handlers.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth import get_user_model
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from guardian.shortcuts import assign_perm
+
+from measurement.models import Measurement, Report
+
+User = get_user_model()
+
+
+@receiver(post_save, sender=Measurement)
+@receiver(post_save, sender=Report)
+def set_permissions(sender, instance, **kwargs):
```

Comment:
We might need to come up with a different plan to limit what data is accessible without assigning permissions at object level. 

Measurements (and Reports) are normally created in bulk, using [`bulk_create`](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#bulk-create) as that is way, way faster than going element by element. However, using that function does not call the `save` method of the object and therefore there is no `post_save` signal. As a consequence, the permissions of those objects will not be updated. 

I'm not entirely sure what the best way forward would be, to be honest.

Reply:
I wonder if we even need to assign permissions to measurements, as any attempt to view/change/delete a measurement could probably just check the permissions of the associated station. We could add a 'view_measurements' permission to the station model to distinguish between permissions to view the station metadata and permissions to view associated measurements.

Reply:
That sounds like a good plan.